scripts/optimize_i7_13700k.sh

```
@@ -0,0 +1,437 @@
+    run_sudo "echo 0 > /sys/devices/system/cpu/intel_pstate/no_turbo"
+    
+    # Set minimum frequency to high for consistent performance
+    echo "Setting minimum CPU frequency for performance..."
+    run_sudo "cpupower frequency-set -d 3400MHz"  # Base clock of i7-13700K
+    
+    echo "Current CPU frequency information:"
+    cpupower frequency-info | grep -E "(current CPU|analyzing|boost state)"
+else
+    echo "Installing cpupower utilities..."
+    sudo apt-get update
+    sudo apt-get install -y linux-tools-common linux-tools-generic linux-tools-$(uname -r)
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)